tests/solana_tests/using.rs

```diff
@@ -2,98 +2,6 @@
 
 use crate::build_solidity;
 
-#[test]
-fn using_for_contracts() {
-    let mut runtime = build_solidity(
-        r#"
-        interface I {
-            function f(int) external;
-        }
-
-        library L {
-            function F(I i, bool b, int n) public {
-                if (b) {
-                    print("Hello");
-                }
-            }
-        }
-
-        contract C {
-            using L for I;
-
-            function test() public {
-                I i = I(address(0));
-
-                i.F(true, 102);
-            }
-        }"#,
-    );
-
-    let data_account = runtime.initialize_data_account();
-    runtime
-        .function("new")
-        .accounts(vec![("dataAccount", data_account)])
-        .call();
-    runtime.function("test").call();
-
-    assert_eq!(runtime.logs, "Hello");
-
-    let mut runtime = build_solidity(
-        r#"
-        interface I {
-            function f1(int) external;
-            function X(int) external;
-        }
-
-        library L {
-            function f1_2(I i) external {
-                i.f1(2);
-            }
-
-            function X(I i) external {
-                print("X lib");
-            }
-        }
-
-        contract foo is I {
-            using L for I;
-
-            function test() public {
-                I i = I(address(this));
-
-                i.X();
-                i.X(2);
-                i.f1_2();
-            }
-
-            function f1(int x) public {
-                print("x:{}".format(x));
-            }
-
-            function X(int) public {
-                print("X contract");
-            }
-        }"#,
-    );
-
-    let data_account = runtime.initialize_data_account();
-    runtime
-        .function("new")
-        .accounts(vec![("dataAccount", data_account)])
-        .call();
-
-    let program_id = runtime.stack[0].id;
-    runtime
-        .function("test")
-        .accounts(vec![
-            ("I_programId", program_id),
-            ("systemProgram", [0; 32]),
-        ])
-        .call();
-
-    assert_eq!(runtime.logs, "X libX contractx:2");
-}
-
```

Comment:
I think you've deleted the only runtime test for `using` on Solana. It should be re-written